requirements.txt

```diff
@@ -1,24 +1,29 @@
 alembic==1.8.1
 ascii_magic==2.3.0
--e git+https://github.com/abrichr/atomacos.git; sys_platform == 'darwin'
+atomacos @ git+https://github.com/abrichr/atomacos.git#egg=atomacos ; sys_platform == 'darwin'
 bokeh==2.4.3
 clipboard==0.0.4
 deepdiff[optimize]==6.3.0
 dictalchemy3==1.0.0
+drawsvg==2.1.1
 fire==0.4.0
+gpt4all
 ipdb==0.13.11
 loguru==0.6.0
 matplotlib==3.6.2
 mss==6.1.0
+numpy==1.24.3
```

Comment:
Is there a particular reason this is here? What happens if you remove it and try `pip install -r requirements.txt`?